androidApp/src/main/java/com/EENX15_22_17/digital_journal/android/ui/components/Checkbuttons.kt

```diff
@@ -97,44 +99,110 @@ fun <E : Enum<*>> EnumCheckBoxLazyGrid(
     currentSelected: Set<E>,
     labels: Map<E, String>,
     gridLayout: GridCells,
-    circleBgColor: Color=checkBoxColor,
-    textFontSize: Int=20
+    circleBgColor: Color = checkBoxColor,
+    textFontSize: Int = 20,
```

Comment:
This should probably be specified in `.sp`:s with the type `TextUnit`s, since most of our theming specify sizes in that way. (I see that this is not introduced in this commit though)